### ISSUE TITLE
Wrongly classified as non kanken kanji

### ISSUE THREAD

**Issue by Lyroxide** [Kuuuube/kanjigrid#29]
So far I found these that are classified under "Non-Kanji Kentei"

𣶏 (should be 1k)
𬵪 (should be 1k)
𩶗 (should be 1k)
𫒼 (should be 1k)
頹 (should be 1k, 許容字体 for 頽)

Also there's supposedly 2799 kanji for 1k, while there's only 2791 in kanji grid. I guess you should just scrape again from jitenon.

**Comment by Kuuuube** [comment]
The data for this was taken from jitenon by the creator of this anki deck: https://ankiweb.net/shared/info/1833474130. The deck claims this `This deck contains all Kanji of the 日本漢字能力検定 [Kanji Kentei] according to [jitenon.jp](https://jitenon.jp/) (as of 2023).` Maybe the data in kanjigrid is out of date or maybe the deck is wrong? I'm not sure.

Where are you getting the 2799 number from? I'm looking at this page (https://kanji.jitenon.jp/cat/kyu01) on jitenon and it shows only 2797 kanji.

This looks fairly easy to fix if the data on those pages from jitenon are correct.

**Comment by Kuuuube** [comment]
Diff of the removed and added characters:
```
Kanji not in new version:
欝
屮

Kanji not in old (current) version:
戶
成
芽
周
𦤶
彥
姬
城
產
淸
脫
稅
割
窗
隆
銳
遼
輸
諭
𠮟
䇳
喻
彚
𥡴
𨻶
屮
偷
𣶏
頹
𩶗
𫒼
櫸
𬵪
鬭
```

Many characters have had their levels changed so those changes will be made as well.

**Comment by Kuuuube** [comment]
Fixed with e95252e.

**Comment by Lyroxide** [comment]
It is indeed 2797. Thanks for the fix!
